components/public-api-server/pkg/apiv1/workspace.go

```diff
@@ -229,7 +243,13 @@ func (s *WorkspaceService) StartWorkspace(ctx context.Context, req *connect.Requ
 		return nil, proxy.ConvertError(err)
 	}
 
-	workspace, err := convertWorkspaceInfo(ws)
+	authProviders, err := conn.GetAuthProviders(ctx)
```

Comment:
What stops us from returning this is part of `conn.GetWorkspace` from `server`?

Reply:
@easyCZ Not much but types from protocol are also used inside workspace db implementation so I'll need to split them, db types and protocol types, does that sounds good?